src/test/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImplTest.java

```diff
@@ -444,6 +444,28 @@ public void hiddenFieldSet() throws Exception {
         session.save();
     }
 
+    @Test
+    public void hiddenFieldSetUsingNewNameForTemplateMixin() throws Exception {
+        textNode.addMixin("jmix:hiddenFieldSetV2");
+        session.save();
+        EditorForm form = editorFormService.getEditForm(Locale.ENGLISH, Locale.ENGLISH, textNode.getPath());
+        EditorFormFieldSet fs = getFieldSet(form, "content", "jmix:hiddenFieldSetV2");
+
+        Assert.isTrue(!fs.getDisplayed(), "FieldSet should not be displayed");
+        Assert.isTrue(fs.getActivated(), "FieldSet should be activated");
+        Assert.isTrue(!fs.getDynamic(), "FieldSet should not be dynamic");
+
+        fs = getFieldSet(form, "content", "jmix:hiddenFieldSetExtendsV2");
+
+        Assert.isTrue(!fs.getDisplayed(), "FieldSet should not be displayed");
+        Assert.isTrue(!fs.getActivated(), "FieldSet should not be activated");
+        Assert.isTrue(fs.getDynamic(), "FieldSet should be dynamic");
+        // clean up
+
+        textNode.removeMixin("jmix:hiddenFieldSetV2");
+        session.save();
+    }
+
```

Comment:
Factorise this code to use parameters and avoid code copy/pasted with existing hiddenFieldSet()

Reply:
I guess there is no problem with duplicating tests because it's always good to have the right code for each test case. But if you wish, I will update it.